src/queries/hooks/channels/useGetChannelDetails.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+import { useSelector } from 'react-redux';
+
+import { channelDetails } from 'queries/queryKeys';
+import { getChannelDetails } from 'queries/services';
+
+export const useGetChannelDetails = (address: string) => {
+  const { userPushSDKInstance } = useSelector((state: any) => {
+    return state.user;
+  });
```

Comment:
```suggestion
  const { userPushSDKInstance } = useSelector((state: any) => state.user);
```

@abhishek-01k provide proper type to state variable